docs/fundamentals/runtime-libraries/snippets/System/Double/Overview/csharp/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿class Program
+{
+    static void Main(string[] args)
+    {
+        //Example1.Main();
+        //Example2.Main();
+        //Example3.Main();
+        //Example4.Main();
+        //Example5.Main();
+        //Example6.Main();
+        //Example7.Main();
+        //Example8.Main();
+        //Example9.Main();
+        //Example10.Main();
```

Comment:
I might uncomment all these before you merge:

```suggestion
        Example1.Main();
        Example2.Main();
        Example3.Main();
        Example4.Main();
        Example5.Main();
        Example6.Main();
        Example7.Main();
        Example8.Main();
        Example9.Main();
        Example10.Main();
```